charms/kserve-controller/src/templates/serving_runtimes_manifests.yaml.j2

```diff
@@ -20,16 +23,21 @@ spec:
         memory: 2Gi
   protocolVersions:
   - v1
+  - v2
   supportedModelFormats:
   - autoSelect: true
     name: lightgbm
+    priority: 1
     version: "3"
 ---
 apiVersion: serving.kserve.io/v1alpha1
 kind: ClusterServingRuntime
 metadata:
   name: kserve-mlserver
 spec:
+  annotations:
+    prometheus.kserve.io/path: /metrics
```

Comment:
do you know how these work? I think these might be configurations for how prometheus discovers and monitors things, but we get this from the cos stack instead. I might be misunderstanding it though

Reply:
(same comment for the other prometheus entries)

Reply:
Same reply above

Reply:
See [this comment](https://github.com/canonical/kserve-operators/pull/191/files#r1399779471)